org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/FDialog.java

```diff
@@ -51,7 +51,7 @@
  * @date    Feb 25, 2007
  * @version $Revision: 0.10 $
  */
-
+@Deprecated(since="10", forRemoval=true)
```

Comment:
I think it help to point out that this is replace by the Dialog class.

Reply:
Done